### PR TITLE
302 - Add token logo in contract summary page

### PR DESCRIPTION
**Pull request by stevenhansel**
Screen Shot 2022-11-09 at 22 29 56

I added the token logo in the collateral section. I also added some information about the token to make it not empty. I'm open for discussion on other solutions for the placement of the token logo itself.

### CODE REVIEW

**Comment by vantessel** [comment]
Looks good but now we are showing the same info three times. I would show the symbol + top detail ("cUSDC") next to the amounts 

We can even make it clickable linking to the explorer, e.g. https://solscan.io/token/993dVFL2uXWYeoXuEBFXR4BijeXdTv4s6BzsCjJZuwqk

**Comment by stevenhansel** [comment]
> https://solscan.io/token/993dVFL2uXWYeoXuEBFXR4BijeXdTv4s6BzsCjJZuwqk

This is a really good comparison. I'll make the changes ASAP

**Comment by stevenhansel** [comment]
Screen Shot 2022-11-14 at 17 45 27

This is the latest update, let me know what you think, thanks @vantessel

**Comment by vantessel** [comment]
Looks good!

**Comment by vantessel** [comment]
only thing, can we pls make it clickable to the explorer? The url can have the form of https://solscan.io/token/{token_mint}, e.g. https://solscan.io/token/993dVFL2uXWYeoXuEBFXR4BijeXdTv4s6BzsCjJZuwqk for cUSDC @stevenhansel

**Comment by stevenhansel** [comment]
> only thing, can we pls make it clickable to the explorer? The url can have the form of [https://solscan.io/token/{token_mint}](https://solscan.io/token/%7Btoken_mint%7D), e.g. https://solscan.io/token/993dVFL2uXWYeoXuEBFXR4BijeXdTv4s6BzsCjJZuwqk for cUSDC @stevenhansel

Sure, please check it out again @vantessel

**Comment by vantessel** [comment]
lgtm!

File: src/components/molecules/CoinBadge/CoinBadge.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+	amount: number;
+	token: TokenInfo;
+};
+
+const CoinBadge = (props: Props) => {
+	return (
+		<div className={styles.container}>
+			<p className={styles.title}>{props.title}</p>
+
+			<a href={`${solscan.baseUrl}/token/${props.token.address}`} target="_blank" rel="noopener noreferrer" className={styles.chip}>
```
Comment (thegreat-vanderlinde) [inline src/components/molecules/CoinBadge/CoinBadge.tsx:18]: Can we use the library `@vyper-protocol/explorer-link-helper`? It's already in place somewhere in the project. The library currently supports `tx` and `account`, in this case we can treat the mint address as an account. 

Apparently on solscan the following urls redirect to the same:
- https://solscan.io/token/993dVFL2uXWYeoXuEBFXR4BijeXdTv4s6BzsCjJZuwqk
- https://solscan.io/account/993dVFL2uXWYeoXuEBFXR4BijeXdTv4s6BzsCjJZuwqk

We can harcode solscan as the preferred explorer (maybe in the future we can make it selectable as a user preference) and we can inject the current cluster from the `env`.

**Review by thegreat-vanderlinde — CHANGES_REQUESTED** [review]
overall looks like a great implementation, can you just please fix the url as per the comment below?

File: src/components/molecules/CoinBadge/CoinBadge.tsx (hunk 1 above)
Comment (stevenhansel) [reply src/components/molecules/CoinBadge/CoinBadge.tsx]: Done! @thegreat-vanderlinde please check my newest commit, thanks!

**Review by thegreat-vanderlinde — APPROVED** [review]
perfect!

**Pull request merged**